knowledge_repo/app/models.py

```diff
@@ -275,7 +309,7 @@ def get_subscriptions(self):
         return out_subscriptions
 
     @property
-    def get_liked_posts(self):
+    def liked_posts(self):
```

Comment:
[nit] Changing the name of this function seems unrelated to this diff - for the future, can we separate renames from core functionality (this diff is really large as is).